Doc/library/datetime.rst

```diff
@@ -29,8 +29,9 @@ is used to represent a specific moment in time that is not open to
 interpretation [#]_.
 
 A naive object does not contain enough information to unambiguously locate
-itself relative to other date/time objects.  Whether a naive object represents
-Coordinated Universal Time (UTC), local time, or time in some other timezone is
+itself relative to other date/time objects.  Native objects are no longer
```

Comment:
Was this an intentional change? It looks like you kept in the beginnings of an explanation of the problem, but also the problematic old text.

This is actually a bit complicated to reword, because the sentiment behind it has changed, and we may need to hedge a bit. Let me think about how best to communicate the new intent....